src/go/k8s/pkg/resources/certificates.go

```diff
@@ -0,0 +1,91 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+const (
+	// operatorClientCert cert name - used by kubernetes operator to call KafkaAPI
```

Comment:
Is there a technical motivation in moving the const cert names away from the corresponding files? I don't actually mind it, just curious.